modules/monitoring-modifying-the-retention-time-for-thanos-ruler-metrics-data.adoc

```diff
@@ -0,0 +1,68 @@
+// Module included in the following assemblies:
+//
+// * monitoring/configuring-the-monitoring-stack.adoc
+
+:_content-type: PROCEDURE
+[id="modifying-the-retention-time-for-thanos-ruler-metrics-data_{context}"]
+= Modifying the retention time for Thanos Ruler metrics data
+
+By default, for user-defined projects Thanos Ruler automatically retains metrics data for 24 hours.
+You can modify the retention time to change how long this data is retained by specifying a time value in the `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` namespace.
```

Comment:
This sentence is a little on the long side. If you think it would help the flow, consider breaking it up into two sentences. Note, my suggestion actually _adds_ word count, so only make the change if you think two sentences help readability and understandability :) 

`You can modify the retention time to change how long this data is retained. To make this modification, specify  a time value in the `user-workload-monitoring-config` config map in the `openshift-user-workload-monitoring` namespace.`

Reply:
I think it works better as a single sentence -- fewer words overall. Leaving as-is.